message-service/build.gradle.kts

```
@@ -202,4 +203,10 @@ tasks {
         classpath = ktlint
         args = listOf("applyToIDEAProject", "-y")
     }
+
+    dependencyCheck {
+        failBuildOnCVSS = 0.0f
+        analyzers.assemblyEnabled = false
```

In my experience, it's best to disable **all** the unused analyzers as the plugin will spend some unnecessary time initializing and attempting to find those non-existent dependencies:
```suggestion
        analyzers(delegateClosureOf<org.owasp.dependencycheck.gradle.extension.AnalyzerExtension> {
            assemblyEnabled = false
            nodeAuditEnabled = false
            nodeEnabled = false
            nuspecEnabled = false
        })
```